docs/src/sections/sprints/sprint_9.adoc

```diff
@@ -1,5 +1,103 @@
+|LP-129|Create server url input view|Jan Pieczul|3|Ukończone
+|LP-125|Create upload keys view|Jan Pieczul|3|Ukończone
+|LP-126|Create notification list view|Aleksander Konieczny|3|Ukończone
+|LP-132|Add notification details component|-|2|Nieukończone
+|LP-127|Add confirmation/denial flow|-|5|Nieukończone
+|LP-133|Handle push notifications|-|3|Nieukończone
+|LP-47|Create readme for webservice|Aleksander Konieczny|1|Ukończone
+|LP-79|Implement user details controller|Aleksander Konieczny|2|Ukończone
+|LP-116|Create form for creating new transactions|Aleksander Konieczny|2|Ukończone
+|LP-141|Write "Plan" chapter|Jan Pieczul|3|Nieukończone
+|LP-151|Write "Proces realizacji" chapter - part 1|Sebastian Lewandowski|8|Nieukończone
+|LP-160|Implement wallet recreation flow|-|3|Nieukończone
+|LP-135|Create auth scenarios|Sebastian Lewandowski|2|Nieukończone
+|LP-136|Create payment scenarios|Sebastian Lewandowski|3|Nieukończone
+|LP-138|Create transaction scenarios|Sebastian Lewandowski|3|Nieukończone
+|LP-162|Add generic way to send tokens to downstream systems|-|3|Nieukończone
+|LP-104|Handle payments from non-logged users|Jan Pieczul|3|Nieukończone
+|===
+
+Zaplanowane zostało 17 zadań wartych 49 punktów historii. Większość z tych zadań była już wcześniej planowana w
+innych iteracjach. W trakcie sprintu dodane zostały 2 zadania warte 5 punktów. Ostatecznie zakres zadań wyniósł
+19 zadań wartych 54 punkty. Pomimo tego, że termin prac przesunięto na początek września, aplikacja mobilna wciąż
+musiała zostać ukończona w czerwcu. Z tego powodu w tej iteracji ponownie zawarto wiele zadań.
+
 ==== Przebieg prac
 
+Priorytetem tej iteracji był rozwój aplikacji mobilnej. Na początku praz napotkano kilka poważnych problemów.
+Okazało się, że biblioteka obsługująca protokół Bitcoina w prototypie nie jest kompatybilna z aplikacją mobilną
+stworzoną z zestawem narzędzi _Expo_. Było to bardzo problematyczne, gdyż zablokowało to rozwój aplikacji. Zespół
+musiał znaleźć rozwiązanie pozwalające na jej użycie. Zdecydowano się na pozbycie się _Expo_. Praktycznie
+cała aplikacja musiała zostać przepisana, gdyż proces tworzenia oprogramowania tylko i wyłącznie z wykorzystaniem
+narzędzi dostarczanych z _React Native_ kompletnie różni się od _Expo_. Po długiej walce udało się przenieść aplikację
+na domyślne narzędzia budowania. Udało się także dodać bibliotekę _BitcoinJS_, dzięki zastosowaniu narzędzia
+_rn-nodeify_, pozwalającego na dodanie pakietów _Node_ do aplikacji tworzonej w _React Native_. Kolejnym problemem
+okazał się wcześniej dodany pakiet wspólny _common-ts_. Miał on na celu pozbycie się duplikatów z aplikacji
+internetowej oraz mobilnej. Niestety praca z tym pakietem okazała się bardzo nieporęczna. Wymagana była jego
+kompilacja oraz ręczne dodanie przy każdej modyfikacji, co przy równoczesnej pracy nad innymi komponentami bardzo
+utrudniało programowanie. Nie istnieje rozwiązanie pozwalające na dynamiczną aktualizację pakietów, więc zdecydowano
```

Comment:
czas terazniejszy

Reply:
też zaplanowane, bo dalej to rozwiązanie na to pozwalające i nic się do dzisiaj nie zmieniło